src/models/observation.js

```diff
@@ -79,17 +79,17 @@ module.exports = (sequelize, DataTypes) => {
     result: { type: DataTypes.JSONB, allowNull: false },
     resultTime: {
       type: DataTypes.DATE,
-      allowNull: false,
+      allowNull: true,
```

Comment:
Add a comment here explaining what the spec says:
`Note: Many resource-constrained sensing devices do not have a clock. As a result, a client may omit resultTime when POST new Observations, even though resultTime is a mandatory property. When a SensorThings service receives a POST Observations without resultTime, the service SHALL assign a null value to the resultTime.`